### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by JimGaleForce**
specific changes to allow appx bundling

### CODE REVIEW

File: SpeechTranslator/SpeechTranslator.csproj [hunk 1]
```diff
@@ -146,6 +147,9 @@
   <PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
```
Comment (bfjelds) [inline SpeechTranslator/SpeechTranslator.csproj:151]: why is this added?

Comment (JimGaleForce) [inline SpeechTranslator/SpeechTranslator.csproj:151]: required for appx deployment

**Comment by JimGaleForce** [comment]
reviewed by turkycat and herrick
